server/migrations/2021-04-03-161950_add_dates/up.sql

```diff
@@ -0,0 +1,33 @@
+ALTER TABLE users
+    ADD created timestamp NOT NULL default current_timestamp,
+    ADD updated timestamp NOT NULL default current_timestamp;
+
+ALTER TABLE rooms
+    ADD created timestamp NOT NULL default current_timestamp,
+    ADD updated timestamp NOT NULL default current_timestamp;
+
+ALTER TABLE rooms_users
+    ADD created timestamp NOT NULL default current_timestamp,
+    ADD updated timestamp NOT NULL default current_timestamp;
+
+CREATE FUNCTION last_upd_trig() RETURNS trigger
+    LANGUAGE plpgsql AS
+$$BEGIN
+    NEW.updated := current_timestamp;
+    RETURN NEW;
+END;$$;
+
+CREATE TRIGGER user_update_date_trigger
+    BEFORE UPDATE ON users
+    FOR EACH ROW
+    EXECUTE PROCEDURE last_upd_trig();
+
+CREATE TRIGGER room_update_date_trigger
+    BEFORE UPDATE ON rooms
+    FOR EACH ROW
+    EXECUTE PROCEDURE last_upd_trig();
+
+CREATE TRIGGER rooms_updates_update_date_trigger
+    BEFORE UPDATE ON rooms
```

Comment:
This should probably be
```suggestion
    BEFORE UPDATE ON rooms_users
```